src/components/OrderBookChart.stories.tsx

```diff
@@ -0,0 +1,256 @@
+
+const networkIds = Object.values(Network).filter(Number.isInteger)
+
+const defaultNetworkId = Network.Mainnet
+// All Default Tokens
+const tokenList: TokenDetails[] = CONFIG.initialTokenList.map(
+  ({ id, name, symbol, addressByNetwork, decimals = 18 }) => ({
+    id,
+    name,
+    symbol,
+    address: addressByNetwork[defaultNetworkId] || '0x',
+    decimals: decimals,
+  }),
+)
+
+// Default params to be used as initial values
+// and when there's no Token found for symbol
+const defaultParams: Omit<OrderBookChartProps, 'data'> = {
+  baseToken: {
+    id: 1,
+    name: 'Base Token',
+    symbol: 'BASE',
+    address: '0x1',
+    decimals: 18,
+  },
+  quoteToken: {
+    id: 2,
+    name: 'Quote Token',
+    symbol: 'QUOTE',
+    address: '0x2',
+    decimals: 18,
+  },
+  networkId: defaultNetworkId,
+}
+
+// Token symbols to use in control selector
+const configTokenSymbols = [defaultParams.baseToken, defaultParams.quoteToken, ...tokenList].map(
+  (token) => token.symbol || token.address,
+)
+
+export default {
+  title: 'Components/OrderBookChart',
+  component: OrderBookChart,
+  argTypes: {
+    networkId: { control: { type: 'inline-radio', options: networkIds } },
+    baseToken: {
+      control: { type: 'select', options: configTokenSymbols },
+    },
+    quoteToken: {
+      control: { type: 'select', options: configTokenSymbols },
+    },
+    data: { control: null },
+  },
+  decorators: [(Story): JSX.Element => <div style={{ height: '97vh' }}>{Story()}</div>],
+} as Meta
+
+//-------------------REAL-DATA------------------
+
+// Template for real data
+// fetched from price-estimation service
+const RealTemplate: Story<OrderBookChartProps> = (args) => <OrderBookChart {...args} />
+
+const DAI = tokenList.find((token) => token.symbol === 'DAI')
+const USDC = tokenList.find((token) => token.symbol === 'USDC')
+export const USDC_DAI = RealTemplate.bind({})
+USDC_DAI.args = {
+  ...defaultParams,
+  baseToken: DAI || tokenList[0],
+  quoteToken: USDC || tokenList[1],
+  data: processRawApiData({ data: realData, baseToken: DAI || tokenList[0], quoteToken: USDC || tokenList[1] }),
+}
+USDC_DAI.argTypes = {
+  baseToken: { control: null },
+  quoteToken: { control: null },
+  data: { control: null },
+}
+
+//-------------------SAMPLE-DATA------------------
+
+interface ProduceOrderBookArgsParams {
+  sampleData: Pick<SampleData, 'asks' | 'bids'>
+  orderbookProps: Omit<OrderBookChartProps, 'data'>
+}
+// normalizing sample data
+// because real data is slightly more complex
+const produceOrderBookProps = ({ sampleData, orderbookProps }: ProduceOrderBookArgsParams): OrderBookChartProps => {
+  const { baseToken, quoteToken } = orderbookProps
+
+  const { asks = [], bids = [] } = sampleData
+  const normalizedRawData: RawApiData = {
+    asks: asks.map(({ amount, price }) => ({
+      // turn small numbers into wei or decimal-aware units
+      // otherwise get filtered as too small by src/components/OrderBookWidget.tsx#L77
```

Comment:
You know this comment will be outdated in the next change to OrderBookWidget file, right?